app/models/runtime/organization.rb

```diff
@@ -164,13 +164,7 @@ def add_user(user)
 
     def self.user_visibility_filter(user)
       {
-        id: dataset.join_table(:inner, :organizations_managers, organization_id: :id, user_id: user.id).select(:organizations__id).union(
-          dataset.join_table(:inner, :organizations_users, organization_id: :id, user_id: user.id).select(:organizations__id)
-        ).union(
-          dataset.join_table(:inner, :organizations_billing_managers, organization_id: :id, user_id: user.id).select(:organizations__id)
-        ).union(
-          dataset.join_table(:inner, :organizations_auditors, organization_id: :id, user_id: user.id).select(:organizations__id)
-        ).select(:id)
+        id: OrganizationRole.where(user_id: user.id).select(Sequel.as(:organization_id, :id))
```

Comment:
Did you find `OrganizationRole.where` to be faster than the joins?  In some of our testing we find UNIONs with the `where` call executed after to be slow, so curious if this doesn't follow that pattern.  I wrote more about it here: https://github.com/cloudfoundry/capi-release/issues/278#issuecomment-1303905237

Reply:
Ahh right I hadn't been aware that you'd observed that being slower, but I guess it makes sense that `OrganizationRole` would result in a bunch of unnecessary role rows being fetched, only to be filtered out with the `where` at the higher level.

I've change this (here and in a couple of other places) to be a union of the individual role tables with separate `where` clauses at the 'lower' level.